### PR TITLE
Add back BLCursor as deprecated

### LINKED ISSUE #403: Regression on BlCursor
Hi, 

All zones of my UI where i change the cursor is broken with latest version of Bloc.

exemple
```myBlSpace currentCursor: BlCursor crosshair```

BlCursor does not exist anymore... it was existent some month ago.
Thanks by advance.

Eric

### PR DESCRIPTION
**Pull request by jecisc**
Fixes #403 

Sorry for the inconvenance

### CODE REVIEW

File: src/Bloc/BLCursor.class.st [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	#superclass : 'Object',
+	#category : 'Bloc-Utilities - Selection',
+	#package : 'Bloc',
+	#tag : 'Utilities - Selection'
+}
+
+{ #category : 'accessing' }
+BLCursor class >> crosshair [
+
+	self deprecated: 'Use Cursor instead.' transformWith: '`@rcv crosshair' -> 'Cursor crosshair'.
```
Comment (tinchodias) [inline src/Bloc/BLCursor.class.st:12]: I think it must be `crossHair` (uppercase H)

**Comment by tinchodias** [comment]
Thanks Cyril, it was my responsibility before merging to realize a deprecation was needed, but i merged too fast!

File: src/Bloc/BLCursor.class.st (hunk 1 above)
Comment (jecisc) [reply src/Bloc/BLCursor.class.st]: Great catch thanks

**Comment by jecisc** [comment]
This should be good now

**Comment by tinchodias** [comment]
Thanks @jecisc

**Comment by tinchodias** [comment]
It was BlCursor, with Bl not BL :) I'm pushing the rename :)

**Pull request merged**